packages/deployments/contracts/contracts/messaging/RootManager.sol

```diff
@@ -36,6 +36,8 @@ contract RootManager is ProposedOwnable, IRootManager, WatcherClient, DomainInde
 
   event ConnectorRemoved(uint32 domain, address connector, uint32[] domains, address[] connectors, address caller);
 
+  event Log(string message);
```

Comment:
Propagate error event maybe?

Reply:
`event Event`

Reply:
would call it `PropogateFailed` or something? idk

Reply:
something more specific

Reply:
bump

Reply:
resolved